src/api/app/controllers/webui/package_controller.rb

```diff
@@ -390,8 +393,9 @@ def set_file_details
       end
     rescue ActiveXML::Transport::Error => e
       # TODO crudest hack ever!
-      if e.summary == 'service in progress'
+      if e.summary == 'service in progress' && @expand == 1
         @expand = 0
+        @service_running = true
```

Comment:
``set_file_details`` is a helper method which gets called in the ``show`` action, see line ``68``.